src/components/Reports/GoalCalculator/SummaryReport/Steps/PresentingYourGoalStep/PresentingYourGoal.tsx

```diff
@@ -0,0 +1,400 @@
+import React, { useMemo, useRef } from 'react';
+import PrintIcon from '@mui/icons-material/Print';
+import {
+  Box,
+  Button,
+  Divider,
+  Grid,
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableRow,
+  Typography,
+  styled,
+  useMediaQuery,
+} from '@mui/material';
+import { useTheme } from '@mui/material/styles';
+import { useTranslation } from 'react-i18next';
+import {
+  Cell,
+  Legend,
+  Pie,
+  PieChart,
+  ResponsiveContainer,
+  Tooltip,
+} from 'recharts';
+import { useGetUsersOrganizationsAccountsQuery } from 'src/components/Settings/integrations/Organization/Organizations.generated';
+import { useGetUserQuery } from 'src/components/User/GetUser.generated';
+import { useLocale } from 'src/hooks/useLocale';
+import { useOrganizationId } from 'src/hooks/useOrganizationId';
+import cruLogo from 'src/images/cru/cru.svg';
+import { currencyFormat } from 'src/lib/intlFormat';
+
+const ChartContainer = styled(Box)(({ theme }) => ({
+  width: '70%',
+  height: 500,
+
+  [theme.breakpoints.down('lg')]: {
+    width: '100%',
+  },
+
+  '@media print': {
+    width: '100% !important',
+    height: '350px !important',
+    '.recharts-wrapper': {
+      height: '100px !important',
+      width: '100% !important',
+    },
+    '.recharts-surface': {
+      height: '350px !important',
+      width: '100% !important',
+    },
+    '.recharts-legend-wrapper': {
+      fontSize: '16px !important',
+      paddingRight: '20px !important',
+      textAlign: 'center !important',
+    },
+  },
+
+  '& .recharts-legend-item .recharts-surface': {
+    width: '16px !important',
+    height: '16px !important',
+    top: '2px',
+  },
+
+  '& .recharts-legend-item text': {
+    dominantBaseline: 'middle',
+  },
+}));
+
+const StyledTableCell = styled(TableCell)({
+  border: 'none',
+  paddingTop: 16,
+  paddingBottom: 16,
```

Comment:
Can we use spacing values from the theme? You can also use `paddingBlock` to set the same padding for top and bottom.